spec/controllers/users_controller_spec.rb

```diff
@@ -67,6 +75,42 @@
       end
     end
 
+    context "recommended by existing user" do
+      let(:existing_user) { Fabricate(:user) }
+      let(:user_params) { Fabricate.attributes_for(:user) }
+      let(:new_user) { User.second }
+      let(:invitation) { Fabricate(:invitation, inviter: existing_user, token: "token") }
+
+      it "sets new user to as a follower of the existing user" do
+        post :create, params: { user: user_params, token: invitation.token }
+        expect(new_user.follows? existing_user).to be
+      end
+
+      it "sets existing user as a follower of the new user" do
+        post :create, params: { user: user_params, token: invitation.token }
+        expect(existing_user.follows? new_user).to be
+      end
+
+      it "deletes token from existing user" do
+        post :create, params: { user: user_params, token: invitation.token }
+        expect(invitation.reload.token).to be_nil
+      end
+
+      it "sets a flash message notifying the newly registered user that they are following their friend" do
```

Comment:
This is another one where you should test the message if your spec title specifies what it should be.